KIF.podspec

```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.license                 = 'Apache 2.0'
   s.authors                 = 'Michael Thole', 'Eric Firestone', 'Jim Puls', 'Brian Nickel'
   s.source                  = { :git => "https://github.com/kif-framework/KIF.git", :tag => "v#{ s.version.to_s }" }
-  s.platform                = :ios, '8.0'
+  s.platform                = :ios, '9.0'
```

Comment:
Should this be 11, given that is the minimum supported version by the library?

Reply:
Followup issue here: https://github.com/kif-framework/KIF/issues/1177